client/ayon_kitsu/hooks/pre_status_cahnge.py

```diff
@@ -0,0 +1,101 @@
+from ayon_applications import PreLaunchHook
+from ayon_kitsu.addon import is_kitsu_enabled_in_settings
+
+import os
+import gazu
+import re
+
+
+
+class PreStatusChange(PostLaunchHook):
+    order = 1
+    launch_types = ()
```

Comment:
Pretty sure this should be an empty set instead.

```suggestion
    launch_types = set()
```